src/test/java/uk/gov/pay/api/validation/DirectDebitEventSearchValidatorTest.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.pay.api.validation;
+
+
+import org.junit.Test;
+import uk.gov.pay.api.exception.ValidationException;
+
+import java.time.ZonedDateTime;
+
+import static org.junit.Assert.assertTrue;
+
+public class DirectDebitEventSearchValidatorTest {
+    
+    
+    @Test
+    public void shouldPassValidation() {
+        DirectDebitEventSearchValidator.validateSearchParameters(ZonedDateTime.now().toString(), ZonedDateTime.now().toString(), 1);
+    }
+
+    @Test(expected = ValidationException.class)
+    public void shouldFailWithInvalidDisplaySize() {
+        try {
+            DirectDebitEventSearchValidator.validateSearchParameters(ZonedDateTime.now().toString(), ZonedDateTime.now().toString(), 501);
+        } catch (ValidationException ex) {
+            assertTrue(ex.getPaymentError().getDescription().contains("display_size"));
+            throw ex;
```

Comment:
Another way to do this would be to remove the `expected = ValidationException.class`, not rethrow the exception, and add a `fail()` call after the `catch` block. But this works!